cluster/gce/coreos/node-rkt.yaml

```diff
@@ -53,7 +53,7 @@ coreos:
         RemainAfterExit=yes
         EnvironmentFile=/etc/kube-env
         ExecStartPre=/usr/bin/mkdir -p /etc/rkt /opt/downloads /opt/rkt/
-        ExecStartPre=/usr/bin/curl --fail --silent --location --create-dirs --output /opt/downloads/rkt.tar.gz https://github.com/coreos/rkt/releases/download/v${RKT_VERSION}/rkt-v${RKT_VERSION}.tar.gz
+        ExecStartPre=/usr/bin/curl --fail --silent --location --create-dirs --output /opt/downloads/rkt.tar.gz https://github.com/coreos/rkt/releases/download/${RKT_VERSION}/rkt-${RKT_VERSION}.tar.gz
```

Comment:
I'd prefer leaving it as before. Mostly for backwards compatibility (though with no releases that matters less), but also since `RKT_VERSION=1.8.0` feels like a more expected value than `v1.8.0`.. if only marginally.